corehq/apps/api/resources/v0_5.py

```diff
@@ -302,6 +303,7 @@ class Meta(WebUserResource.Meta):
         authentication = AdminAuthentication()
         detail_allowed_methods = ['get']
         list_allowed_methods = ['get']
+        throttle = BaseThrottle()
```

Comment:
If the intention of `AdminResourceMeta` is that any admin Meta inherits from it, this Meta is now pretty confusing. It's a class that begins with 'Admin', is listed in `ADMIN_API_LIST`, but doesn't inherit from `AdminResourceMeta`?  I'd imagine this _could_ inherit from `AdminResourceMeta` with no issues, but for the sake of getting this out, I can understand not experimenting with it -- I'd just want to add some kind of comment explaining that why it isn't inheriting from `AdminResourceMeta`.

Reply:
Do you mean to inherit from both `AdminResourceMeta` and `WebUserResource.Meta`?

Reply:
I see your point. I don't want to encourage multiple inheritance, and I see what feels like an unnecessary inheritance hierarchy as is (WebUserResource relies on v0_1.WebUserResource, which relies on UserResource, which relies on CustomResourceMeta). The only thing we get out of this inheritance hierarchy that `AdminMeta` doesn't provide is `object_class` and `resource_name`, which don't seem like the kind of thing we need inheritance for.